applications/gafaelfawr/values.yaml

```diff
@@ -275,6 +275,12 @@ config:
   # [DMTN-235](https://dmtn-235.lsst.io/) for more details on scopes.
   groupMapping: {}
 
+  # -- Defines additional FQDNs for Gafaelfawr.  This doesn't work for
+  # cookie or browser authentication, but for token-based services like
+  # git-lfs or the webdav server it does.
+
+  additionalFQDNs: []
```

Comment:
The Kubernetes `Ingress` schema calls this parameter `host` for better or for worse, so let's stay consistent with Kubernetes terminology and call this `additionalHosts`. It should go under a top-level `ingress` key, not under `config`, because it customizes the ingress, not the Gafaelfawr configuration.

Reply:
Done.